libkbfs/backpressure_disk_limiter_test.go

```diff
@@ -420,7 +420,7 @@ func TestDefaultDoDelayCancel(t *testing.T) {
 	cancel()
 
 	err := defaultDoDelay(ctx, individualTestTimeout)
-	require.Equal(t, ctx.Err(), errors.Cause(err))
+	require.Equal(t, context.Canceled, errors.Cause(err))
```

Comment:
Yeap! I think it doesn't even matter that defaultDoDelay waits on ctx.Done(), since cancel() is cancel() happens before defaultDoDelay.